copy/copy.go

```diff
@@ -313,7 +316,7 @@ func Image(ctx context.Context, policyContext *signature.PolicyContext, destRef,
 
 	if !multiImage {
```

Comment:
> converting a non-multi-arch single-image image into a new manifest list with two compression variants

BTW I think this the case necessary for a simple `buildah build && buildah push`. So that will need to happen — but it might be a separate future PR. (Alternatively, buildah could create a single-instance manifest list itself, but that seems unnecessarily redundant.)